release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -216,6 +216,11 @@ For further information about gathering debugging information relating to low-la
 ==== Support for provisioning IPv6 spoke clusters from dual-stack hub clusters
 With this update, you can provision IPv6 address spoke clusters from dual-stack hub clusters. In a zero touch provisioning (ZTP) environment, the HTTP server on the hub cluster that hosts the boot ISO now listens on both IPv4 and IPv6 networks. The provisioning service also checks the baseboard management controller (BMC) address scheme on the target spoke cluster and provides a matching URL for the installation media. These updates offer the ability to provision single-stack, IPv6 spoke clusters from a dual-stack hub cluster. 
 
+[id="ocp-4-14-precaching-user-spec-images"]
+==== Pre-caching user-specified images with {cgu-operator-full}
+
+With this release, you can pre-cache your application workload images before upgrading your applications on {sno} clusters with {cgu-operator-full}. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc#talm-prechache-user-specified-images_ztp-talm[Pre-caching user-specified images with TALM on single-node OpenShift clusters].
```

Comment:
I don't see the section this is linking to - is that section coming in an unmerged PR?

Reply:
Yes, it's unmerged at the moment. I usually make the updates on main and create the RN PRs before merge so I don't have to do it during the release chaos and possibly miss one :)

For now, I just wanted to get the wording peer reviewed! I'll check the link myself once the linked content is merged and I'm sure it'll be checked during the merge review too. I hope that makes sense!